src/main/MainTabsScreen.js

```diff
@@ -52,7 +52,7 @@ export default function MainTabsScreen(props: Props) {
       <OfflineNotice />
       <Tab.Navigator
         {...bottomTabNavigatorConfig({
-          showLabel: !!Platform.isPad,
+          showLabel: Platform.OS === 'ios' && Platform.isPad,
```

Comment:
Yeah; this could be made clearer—I hopped over here from https://github.com/zulip/zulip-mobile/pull/4750#discussion_r636555727 to add to the "flow config: Stop ignoring Android-only JS files" commit, since I realized that that commit might be a good place to add the `module.file_ext=.android.js` line that's apparently needed for `react-native-paper`'s [setup](https://callstack.github.io/react-native-paper/getting-started.html). (And then I could cherry-pick the improved commit over to that other PR if it didn't land here first.) So I wasn't paying close attention to the context here.

Specifically, I get this error on the "Stop ignoring Android-only JS files" commit if I leave it as `!!Platform.isPad`:

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/main/MainTabsScreen.js:55:33

Cannot get Platform.isPad because property isPad is missing in object literal [1]. [prop-missing]

     src/main/MainTabsScreen.js
      52│       <OfflineNotice />
      53│       <Tab.Navigator
      54│         {...bottomTabNavigatorConfig({
      55│           showLabel: !!Platform.isPad,
      56│           showIcon: true,
      57│         })}
      58│         backBehavior="none"

     node_modules/react-native/index.js
 [1] 462│   get Platform(): Platform {



Found 1 error
```

I think the [linked doc](https://reactnative.dev/docs/platform#ispad-ios) is clear that `isPad` is missing on Android, not iOS; it's marked with an 'iOS' tag. It's less clear that, once Flow is made aware that `isPad` is missing on Android (by having it not ignore Android-only JS files), Flow seems to count as unsafe any attempts to access `isPad` on Android, even if they'd basically be harmless. So this a necessary change, unless we want a `$FlowFixMe` or similar. I'll say that in the commit message, and I'll also leave a comment that we should keep an eye out for API changes that can let us know if we're on an Android tablet.